imagestreams/mariadb-rhel7-ppc64le.json

```diff
@@ -1,49 +0,0 @@
-{
```

Comment:
don't delete this file ... and if the 10.3 version of mariadb is available on ppc64le, let's add that as well

Then create a `mariadb-rhel7-s390x.json` file with `"tags": "database, mariadb, s390x"` and include imagestreamtags for each version of mariadb available on s390x